prime-router/metadata/fhir_mapping/hl7/segments/MSH/MessageHeader.yml

```diff
@@ -15,10 +15,31 @@ meta:
   expressionType: resource
   valueOf: datatypes/PT/Meta
   specs: MSH.11
+  vars:
+    metaSecurity: STRING, MSH.8
 
 source:
-  valueOf: secondary/Source
-  expressionType: resource
+  expressionType: nested
+  vars:
+    msh3: STRING_ALL, MSH.3
+  expressions:
+    - constants:
+        messageHeaderSourceField: "MSH.3"
```

Comment:
This eventually populate an hl7-use extension. In other places where we have hl7-use extensions we're using the HL7 field name. So in this case it would be `sending-application` instead of `MSH.3`. There's some pros and cons to both strategies. I feel myself leaning a bit more to the name version but don't feel super strong. What do you think?

Reply:
I also don't feel super strongly and it does feel like it comes out in the wash.  I'll switch to the actual name though so it's consistent.